CHANGELOG.md

```diff
@@ -83,6 +83,11 @@ to now accept a `codec.JSONMarshaler` for modular serialization of genesis state
   - `NewKeyring()` now accepts a new backend: `MemoryBackend`.
   - `New()` has been renamed to`NewLegacy()`, which now returns a `LegacyKeybase` type that only allows migration of keys from the legacy keybase to a new keyring.
 * (client/keys) [\#5889](https://github.com/cosmos/cosmos-sdk/pull/5889) Rename `NewKeyBaseFromDir()` -> `NewLegacyKeyBaseFromDir()`.
+* (crypto/keyring) [\#5904](https://github.com/cosmos/cosmos-sdk/pull/5904) `Keybase` -> `Keyring` interfaces migration. `LegacyKeybase` interface is added in order
```

Comment:
Double-check that previously added yet unreleased changelog entries related to the keybase/keyring still belong in the changelog.

Reply:
Ah yes yes, good catch, thanks!